docs/en/sql-reference/aggregate-functions/parametric-functions.md

```diff
@@ -506,3 +506,254 @@ Solution: Write in the GROUP BY query SearchPhrase HAVING uniqUpTo(4)(UserID) >=
 ## sumMapFiltered(keys_to_keep)(keys, values) {#summapfilteredkeys-to-keepkeys-values}
 
 Same behavior as [sumMap](../../sql-reference/aggregate-functions/reference/summap.md#agg_functions-summap) except that an array of keys is passed as a parameter. This can be especially useful when working with a high cardinality of keys.
+
+## sequenceNextNode {#sequenceNextNode}
+
+Returns a value of next event that matched an event chain.
+
+**Syntax**
+
+``` sql
+sequenceNextNode(direction, base)(timestamp, base_condition, event_column, event1, event2, event3, ...)
+```
+
+**Parameters**
+-   `direction` - Used to navigate to directions.
+    - forward : Moving forward
+    - backward: Moving backward
+
+-   `base` - Used to set the base point.
+    - head : Set the base point to the first event
```

Comment:
What actually use-case of `head` and `tail` is? Why we want not to start from first matched event in chain?

Reply:
End-users want to navigate from the beginning of events, but they couldn't know where is the beginning of events. `first_match` and `last_match` has to know where is the beginning. So the `head` and `tail` options may be useful for them.